doc/examples/plot_circular_elliptical_hough_transform.py

```diff
@@ -128,10 +128,10 @@
 
 # Estimated parameters for the ellipse
 best = result[-1]
```

Comment:
I agree we could try and simplify the repetitive calls to `int()` and `round()`. Unfortunately it is not that straightforward.

(1) For a start, it should be `best[1:5]`, however `best` is a [structured array](http://docs.scipy.org/doc/numpy/user/basics.rec.html) so it cannot be sliced like that.

I tried `best[['yc', 'xc', 'a', 'b']]` as proposed in the documentation, but I still get and "invalid index" error.

(2) Then comes the difference between [`round`](https://docs.python.org/2/library/functions.html#round) and [`np.round`](http://docs.scipy.org/doc/numpy/reference/generated/numpy.around.html):

```python
round(0.5) = 1.0
np.round(0.5) = 0.0
```

Personally, I would favor the Python `round` more than the Numpy one. If you look at the two images in my first post, using `np.round` produces to the old output (`int()`) while `round` corresponds to the proposed modification.

As a trade-off between condensed code and respecting all the constraints, I would suggest:

```python
yc, xc, a, b = map( lambda x: int(round(x)), best.tolist()[1:5] )
```

Let me know if you manage to handle the structured arrays in a better way, or if you prefer `np.round` other the default `round`.